TwitchConnect.js

```diff
@@ -62,74 +70,37 @@ twitchClient.on("message", (channel, tags, message, self) => {
   // Ignore echoed messages.
   if (self) return;
 
-  //Sell Stonks
-  if (
-    message.toLowerCase().includes("corpa sell") ||
-    message.toLowerCase().includes("sell corpa")
-  ) {
-    var streamerName = channel.toString().toLowerCase().substring(1);
-    netWorthMap.set(
-      channel.toString().substring(1),
-      netWorthMap.get(channel.toString().substring(1)) - 0.01
-    );
-    insertCorpa(
-      new MongoClient(
-        process.env.MONGO_URL.concat(channel.toString().substring(1))
-      ),
-      streamerName,
-      {
-        channelId: channel,
-        stockName: "$".concat(channel.toString().substring(1)),
-        stockPrice: netWorthMap.get(channel.toString().substring(1)),
-        //timestamp: new Date().toISOString(),
-        timestamp: Math.round(Date.now() / 1000),
-      }
-    );
-  }
-
-  //Buy Stonks
-  if (
-    message.toLowerCase().includes("corpa buy") ||
-    message.toLowerCase().includes("buy corpa")
-  ) {
+  // Add/Sub corpa value
+  if (keywords.includes(message.toLowerCase())) {
     var streamerName = channel.toString().toLowerCase().substring(1);
-    netWorthMap.set(
-      channel.toString().substring(1),
-      netWorthMap.get(channel.toString().substring(1)) + 0.01
-    );
-    insertCorpa(
-      new MongoClient(
-        process.env.MONGO_URL.concat(channel.toString().substring(1))
-      ),
-      streamerName,
-      {
-        channelId: channel,
-        stockName: "$".concat(channel.toString().substring(1)),
-        stockPrice: netWorthMap.get(channel.toString().substring(1)),
-        //timestamp: new Date().toISOString(),
-        timestamp: Math.round(Date.now() / 1000),
-      }
-    );
+    var buySell;
+    if (message.toLowerCase().includes("sell")) buySell = -0.01;
+    else if (message.toLowerCase().includes("buy")) buySell = 0.01;
+    updateNetworthMap(streamerName, buySell);
+    insertCorpa(streamerName);
   }
 
   //Check Streamer Value
-  if (message.toLowerCase().includes("!corpastonk")) {
+  if (message.toLowerCase().includes("!corpastock")) {
     var clientName = new MongoClient(
```

Comment:
Create the client in the `getCorpa` function

Reply:
moved client to getCorpa()